api/src/main/java/io/strimzi/api/kafka/model/common/ContainerEnvVar.java

```diff
@@ -49,6 +52,15 @@ public void setValue(String value) {
         this.value = value;
     }
 
+    @Description("Reference to secret or config map property to which will be the environment variable is set.")
```

Comment:
I see Paul reviewed this before, but it still isn't quite what he proposed and didn't make sense to me, so re-proposing what he suggested.
```suggestion
    @Description("Reference to the secret or config map property to which the environment variable is set.")
```

Reply:
Fixed in #10628